docs/Draft-Generation/using-quote-styles.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Using Quote Styles
+sidebar_position: 3
+slug: /Using-Quote-Styles
+---
+
+# Using Quote Styles
+
+The Quote Style feature automatically restores quotation marks in new drafts so they match the conventions already used in your project. When Scripture Forge generates AI drafts, some punctuation distinctions are lost, this is part of the process and it generally improves the quality of the generated drafts. However, it can create a lot of work for translators who need to restore the distinctions that have been lost. One key distinction is that curly quotation marks are simplified to straight quotes:
+
+'...' (single quotes)
+
+"..." (double quotes)
+
+However, many projects use other styles for quotation marks, such as:
+
+‘ … ’ or “ … ”
+« … »
+「 … 」
+
+On the Generate Draft page is Formatting Options button which will allow you to set up the paragraph breaks and quote styles for your draft. After the initial set up, you can return to the Formatting Options page and change your settings.
+
+![](./paragraph_breaks_image1.png)
+
+You can return to the “Formatting Options” page and change your settings.

```

Comment:
This sentences is repeated (the sentence directly before the screenshot is identical, except for quote marks).